transformer_engine/pytorch/cpp_extensions.py

```diff
@@ -188,6 +188,12 @@ def fused_attn_fwd_qkvpacked(
     if attn_scale is None:
         attn_scale = 1.0 / math.sqrt(d)
 
+    if bias_type != "no_bias":
+        assert (bias is not None
+               and bias.shape == [1, h, max_seqlen, max_seqlen]
+               and bias.dtype == qkv.dtype
+               ), """bias tensor cannot be None when bias_type is not no_bias."""
```

Comment:
It'll be easier for the user to debug if we break this up into three one-line asserts.

Reply:
Done, thanks!